.github/workflows/test.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+name: Init Container Test
+
+on:
+  workflow_call:
+    inputs:
+      INITCONTAINER_LANGUAGE:
+        required: true
+        type: string
+      K8S_OPERATOR_IMAGE_TAG:
+        required: true
+        type: string
+      RUNTIME_VERSION:
```

Comment:
I'm not seeing where this input is being used anywhere in this workflow.

Reply:
I've added a reference in test.yml to pass it to the appropriate step. In practice, I think only Node is going to be using that.